java/core/src/main/java/com/microsoft/bond/BondedBondType.java

```diff
@@ -79,30 +81,35 @@ public final boolean isGenericType() {
 
     @Override
     protected final Bonded<TStruct> newDefaultValue() {
-        return new Bonded<TStruct>(this.valueType.newDefaultValue());
+        // since Bonded is immutable, the default value can be shared
```

Comment:
Okay, I'm fine with this with a clearer comment. How about:
`// Since Bonded<TStruct> provides no API for accessing the TStruct inside it, the default value can be shared.`